.github/workflows/libkineto_cuda.yml

```diff
@@ -0,0 +1,80 @@
+name: libkineto PR Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+# Use TorchBench's docker image which has all basic dependencies.
+env:
+  CONDA_ENV: "torchbench"
+  DOCKER_IMAGE: "ghcr.io/pytorch/torchbench:latest"
+  SETUP_SCRIPT: "/workspace/setup_instance.sh"
+
+jobs:
+  pr-test:
+    # AWS linux instance label: c5.2xlarge
+    runs-on: [linux.2xlarge]
+    timeout-minutes: 180 # 3 hours
+    environment: docker-s3-upload
+    steps:
+      - name: "[FB EMPLOYEES] Enable SSH (Click me for login details)"
+          uses: pytorch/test-infra/.github/actions/setup-ssh@main
+          with:
+            github-secret: ${{ secrets.TORCHBENCH_ACCESS_TOKEN }}
+      - name: Checkout Kineto
+        uses: actions/checkout@v3
+        with:
+          path: kineto
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git submodule sync --recursive
+          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+      - name: Pull docker image
+        uses: pytorch/test-infra/.github/actions/pull-docker-image@main
+        with:
+          docker-image: ${{ env.DOCKER_IMAGE }}
+      - name: Install NVIDIA Driver, docker runtime, set GPU_FLAG
```

Comment:
There is no need to install NVIDIA driver if the instance does not have NVIDIA GPU.